pyproject.toml

```diff
@@ -6,6 +6,5 @@ requires = [
     "decorator",
     "pyyaml",
     "scipy",
-    "vllm==0.7.3",
```

Comment:
I don't think this should be removed. It's used for install from pip.
How about what `Dockerfile.CI` did, add
```
RUN sed -i '/vllm==0.7.3/d' /workspace/vllm-ascend/requirements.txt
RUN sed -i '/[[:space:]]*"vllm==0.7.3",/d' /workspace/vllm-ascend/pyproject.toml
```
in `Dockerfile`